backend/src/main/java/com/soen341/instagram/service/impl/CommentService.java

```diff
@@ -37,15 +38,18 @@
 
 	private static int maxCommentLength = 250;
```

Comment:
I think this should be a constant? If so, also change the name to respect the convention.

Reply:
True, you're right